### ISSUE TITLE
Remove `{{ $frontmatter.description }}` and `{{ $frontmatter.title }}`

### ISSUE THREAD

**Issue by johnnymatthews** [filecoin-project/lotus-docs#6]
The `{{ $frontmatter.description }}` and `{{ $frontmatter.title }}` shortcodes were used throughout the VuePress docs. They're no longer needed since Hugo brings in the `lead` variable into each page, and automatically adds the title to the top of each page.

![image](https://user-images.githubusercontent.com/9611008/138895050-f5ebe8f4-be97-49ea-9b4f-9911c527a81d.png)

```
---
title: "Store and retrieve"
description: "The process of storing and retrieving data using the Filecoin network is slightly different from how most storage platforms work. This tutorial walks you through the whole end-to-end process of keeping your data and then getting it back when you need it! This tutorial should take you about an hour to complete."
lead: "The process of storing and retrieving data using the Filecoin network is slightly different from how most storage platforms work. This tutorial walks you through the whole end-to-end process of keeping your data and then getting it back when you need it! This tutorial should take you about an hour to complete."
draft: false
menu:
    docs:
        parent: "tutorials"
weight: 10
toc: true
mermaid: true
---


## Before we start
```

**Comment by johndmulhausen** [comment]
Done!
